Mathlib/LinearAlgebra/LinearIndependent.lean

```diff
@@ -36,6 +36,9 @@ vector space and `ι : Type*` is an arbitrary indexing type.
 
 * `LinearIndependent R v` states that the elements of the family `v` are linearly independent.
 
+* `LinearIndependentOn R v s` states that the elements of the family `v` indexed by the members
```

Comment:
I wonder if we should save ourself a rename / deprecation later, and use `LinearIndepOn` now. Thoughts, @b-mehta.

Reply:
Yeah, that might be sensible!

Reply:
Done